homeassistant/components/vicare/__init__.py

```diff
@@ -61,8 +61,8 @@ class ViCareRequiredKeysMixin:
                     ): int,  # Ignored: All circuits are now supported. Will be removed when switching to Setup via UI.
                     vol.Optional(CONF_NAME, default="ViCare"): cv.string,
                     vol.Optional(
-                        CONF_HEATING_TYPE, default=DEFAULT_HEATING_TYPE
-                    ): cv.enum(HeatingType),
+                        CONF_HEATING_TYPE, default=DEFAULT_HEATING_TYPE.value
+                    ): vol.In([e.value for e in HeatingType]),
```

Comment:
We normally deprecate the domain key in the config schema with `cv.deprecated` to warn users that the configuration YAML is deprecated.